sdk/src/main/java/io/radar/sdk/RadarSettings.kt

```diff
@@ -118,6 +119,14 @@ internal object RadarSettings {
         getSharedPreferences(context).edit { putBoolean(KEY_TRACKING, tracking) }
     }
 
+    internal fun getListenToServerTrackingOptions(context: Context): Boolean {
+        return getSharedPreferences(context).getBoolean(KEY_SERVER_TRACKING, false)
+    }
+
+    internal fun setListenToServerTrackingOptions(context: Context, tracking: Boolean) {
+        getSharedPreferences(context).edit { putBoolean(KEY_SERVER_TRACKING, tracking) }
```

Comment:
I wonder if we should track when this change happens on the server - or at least log a cloudwatch metric, it'd be great to understand adoption

Reply:
we can probably publish it up in the upsert path

Reply:
added. will add in to the upsert path here